src/resources/extensions/quarto/docusaurus/docusaurus_writer.lua

```diff
@@ -70,10 +70,12 @@ function Writer(doc, opts)
     Callout = function(node)
       local admonition = pandoc.List()
       admonition:insert(pandoc.RawBlock("markdown", ":::" .. node.type))
-      admonition:insert(pandoc.Header(2, node.title))
+      if node.title then
```

Comment:
Is testing coverage a good idea here to protect us from potential future regressions?

Reply:
It would add an additional setup requirement to our test suite, setting up npm, npx, and docusaurus.

We could do it, but I'd like to have a longer discussion first.

Reply:
Is it possible to have quarto only execution without running the npm command for testing? (This may require additional functionality.)

Reply:
It may be worth considering if we have seen and believe we will continue to see these types of regressions in the future.